docs/ja/src/develop/developer.md

```diff
@@ -0,0 +1,9 @@
+# For Developer
+
+**coming soon...**
+
+The following would be added to this section
```

Comment:
```suggestion
```

Reply:
Oh, sorry. I meant to suggest removing an English sentence because there is a Japanese sentence below it that means the same thing. However, I think it is also ok to remove both as you have done.

Reply:
Sorry, I misunderstood.